Cesium.CodeGen/Contexts/IDeclarationScope.cs

```diff
@@ -1,16 +1,18 @@
 using Cesium.CodeGen.Contexts.Meta;
 using Cesium.CodeGen.Ir;
 using Cesium.CodeGen.Ir.Types;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Cesium.CodeGen.Contexts;
 
 internal interface IDeclarationScope
 {
     CTypeSystem CTypeSystem { get; }
-    IReadOnlyDictionary<string, FunctionInfo> Functions { get; }
-    IReadOnlyDictionary<string, IType> Variables { get; }
+    bool TryGetFunctionInfo(string identifier, [NotNullWhen(true)] out FunctionInfo? functionInfo);
```

Comment:
Let's make this to just return `FunctionInfo?`. No need to return `bool` and have this `out` parameter.